### PR TITLE
Frosty04

### PR DESCRIPTION
**Pull request by frosty8104**

### CODE REVIEW

File: src/components/Display.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+function Display() {
```
Comment (mnichols08) [inline src/components/Display.jsx:3]: Maybe we should give this a more descriptive name

**Review by mnichols08 — APPROVED** [review]
I like that you introduced a new component for displaying the objects. I understand that you still need access to the design file in order to implement the design more - and also you require a dataset from whoever is supplying this component with data. I made a suggestion that maybe we should be more specific about the name because it is likely we will want to display other things as well.

**Review by Valeriusdev — CHANGES_REQUESTED** [review]
Please exclude the changes to package-lock.json and package.json from the PR. Also, ensure that the branch deploys successfully before merging it into the development branch.

**Comment by mnichols08** [comment]
This is failing a deploy again after the most recent code push.

We might have to delete this particular package-lock.json file to fix.

```
10:55:18 PM: $ npm run build
10:55:18 PM: > v49-tier2-team-16@0.0.0 build
10:55:18 PM: > vite build
10:55:18 PM: /opt/build/repo/node_modules/vite/node_modules/rollup/dist/native.js:59
10:55:18 PM: 		throw new Error(
10:55:18 PM: 		      ^
10:55:18 PM: Error: Cannot find module @rollup/rollup-linux-x64-gnu. npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). Please try `npm i` again after removing both package-lock.json and node_modules directory.
```

**Review by Valeriusdev — CHANGES_REQUESTED** [review]
Please exclude the changes to package-lock.json and package.json from the PR. Also, ensure that the branch deploys successfully before merging it into the development branch.

**Comment by KseniiaRiabova** [comment]
PR has been closed due to incorrect implementation

**Pull request closed without merging**